core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/rules/auth/BaseGroupsRule.scala

```diff
@@ -69,18 +66,22 @@ abstract class BaseGroupsRule(val settings: Settings)
   override protected def authorize[B <: BlockContext : BlockContextUpdater](blockContext: B): Task[RuleResult[B]] =
     Task.now(RuleResult.Fulfilled(blockContext))
 
+  private def calculateAllowedGroupsForUser(userGroups: UniqueNonEmptyList[Group],
```

Comment:
the first usage of this method is in L111. Can we move it after its first usage?

Reply:
Method removed. It had single, simple usage